v8pp/convert.hpp

```diff
@@ -304,6 +304,53 @@ struct convert<T, typename std::enable_if<std::is_floating_point<T>::value>::typ
 	}
 };
 
+template <typename ... Ts>
+struct convert<std::tuple<Ts...>>
+{
+    using from_type = std::tuple<Ts...>;
+    using to_type = v8::Local<v8::Array>;
+    static constexpr size_t N = sizeof ... (Ts);
+    static bool is_valid(v8::Isolate*, v8::Local<v8::Value> value)
+    {
+        return !value.IsEmpty() && value->IsArray();
+    }
+
+    static from_type from_v8(v8::Isolate * isolate, v8::Local<v8::Value> value)
+    {
+        if (!is_valid(isolate, value))
+        {
+            throw invalid_argument(isolate, value, "Tuple");
+        }
+
+        v8::HandleScope scope(isolate);
+        v8::Local<v8::Context> context = isolate->GetCurrentContext();
+        v8::Local<v8::Array> array = value.As<v8::Array>();
+
+        if (array->Length() != N)
+        {
+            throw std::runtime_error("Invalid array length: expected "
+                                     + std::to_string(N) + " actual "
+                                     + std::to_string(array->Length()));
+        }
+
+        return [isolate, &context, &array]<std::size_t ... Is>(std::index_sequence<Is...>&&) -> from_type{
```

Comment:
As far as I know, lambda template parameters are allowed since C++20. Unfortunately the master branch is in C++14 (or maybe even C++11?), so probably we need here a separate helper function with `index_sequence`, similar to `call_from_v8_impl()`

Reply:
Implemented in 01924ddc04231dec763fa6e014e0d24ea2316a61